security/external_secrets_operator/external-secrets-monitoring.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="external-secrets-monitoring"]
+= Monitoring the External Secrets Operator for Red Hat OpenShift
+include::_attributes/common-attributes.adoc[]
+:context: external-secrets-monitoring
+
+toc::[]
+
+By default, the {external-secrets-operator} exposes metrics for the Operator and the operand. You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.
```

Comment:
nit: should we list the operands? Currently we have just one, but in future we may add couple more.

```suggestion
By default, the {external-secrets-operator} exposes metrics for the Operator and the operands(external-secrets). You can configure OpenShift Monitoring to collect these metrics by using the Prometheus Operator format.
```